src/logPointsManager.ts

```diff
@@ -0,0 +1,420 @@
+var req = require("request");

```

Comment:
These file contains many classes. Since the current code base is pretty flat, we are not sure where to put things. Can you help suggest so we can maybe categorize things into the right folder?

Reply:
Sounds good, addressed in https://github.com/Microsoft/vscode-azureappservice/pull/177 instead.